Rubberduck.Inspections/Inspector.cs

```diff
@@ -135,6 +137,20 @@ private void UpdateInspectionSeverity(Configuration config)
                 return results;
             }
 
+            private static bool RequiredLibrariesArePresent(IInspection inspection, RubberduckParserState state)
+            {
+                var requiredLibraries = inspection.GetType().GetCustomAttributes<RequiredLibraryAttribute>().ToArray();
+
+                if (!requiredLibraries.Any())
+                {
+                    return true;
+                }
+
+                var projectNames = state.DeclarationFinder.Projects.Where(project => project.IsUserDefined == false).Select(project => project.ProjectName).ToArray();
```

Comment:
why not `!project.IsUserDefined` instead of `project.IsUserDefined == false`?

Reply:
We have to write like that at work, so it slipped here as well. Will be corrected.